.github/workflows/storybook-preview.yml

```
@@ -0,0 +1,57 @@
+name: Preview Storybook
+
+on:
+  pull_request:
+    paths:
+      - 'client/**/*.stories.tsx'
+      - 'client/**/*.stories.ts'
+      - 'client/**/*.mdx'
+    types: [opened, reopened, synchronize]
```

진짜 사소하지만 앞의 yml과 통일성을 위해 다음처럼 순서 바꿔도 좋을 것 같아요! 
```suggestion
    types: [opened, reopened, synchronize]
    paths:
      - 'client/**/*.stories.tsx'
      - 'client/**/*.stories.ts'
      - 'client/**/*.mdx'
```